horiz_interp/include/horiz_interp_bicubic.inc

```diff
@@ -450,80 +373,80 @@ module horiz_interp_bicubic_mod
 !    if ( present(fill) ) fill_in = fill
 !   use dfc_x and dfc_y as workspace
 !    if ( fill_in ) call fill_xy(fc(ics:ice,jcs:jce), ics, ice, jcs, jce, maxpass=2)
-!    where ( data_in .le. missing ) data_in(:,:) = 0.
+!    where ( data_in .le. missing ) data_in(:,:) = real(0.0, FMS_HI_KIND_)
```

Comment:
0.0_kindl

Reply:
this is actually commented out (accidental sed victim), I'll delete those few lines of comments